micropub.php

```diff
@@ -925,28 +932,27 @@ public static function store_micropub_auth_response( $args ) {
 	 */
 	public static function store_mf2( $args ) {
 		$props = static::$input['properties'];
-		if ( $props ) {
-			if ( ! isset( $args['meta_input'] ) ) {
-				$args['meta_input'] = array();
-			}
-			$type = static::$input['type'];
+		if ( ! isset( $args['ID'] ) && $props ) {
```

Comment:
why was `! isset( $args['ID'] )` added here? seems unrelated?

Reply:
Because before, it would run the remaining lines, which don't apply on creation, generating an error.

Reply:
When I went to turn it into get, I realized that it was trying to run the replacement code on create.

Reply:
you mean it was running the create code on replace? (and delete and add?) i think that was ok as is, but this is fine too. ok!